src/palace/manager/service/celery/configuration.py

```diff
@@ -31,7 +32,7 @@ class CeleryConfiguration(ServiceConfiguration):
 
     # Broker options for both Redis and SQS
     broker_transport_options_visibility_timeout: int = 3600  # 1 hour
-
+    task_always_eager: bool = False
```

Comment:
In order to override this value in the tests,  we must set it here explicitly even though the default value is False.